spec/ddtrace_integration_spec.rb

```diff
@@ -31,6 +31,11 @@ def thread_count
     end
 
     context 'for file descriptors' do
+      before do
+        # TODO: Restore test for JRuby after further investigation on why it's flaky
+        skip('Flaky test on JRuby. Requires further investigation.') if PlatformHelpers.jruby?
```

Comment:
Minor: Should we leave open an issue (or jira ticket?), and mention it here, so that we don't forget to come back?